### DISCUSSION TITLE
Assigned feature responsibilities

### DISCUSSION THREAD

**Discussion by zlaic** [Zeinab-HasasZiabari/Tafe24S1T102_Vir#6] [Q&A] [answered]
Can we clarify which specific features each team member will be focusing on for this phase of the project?

**Comment by Zeinab-HasasZiabari** [comment, marked as answer]
I am working on "feature_mortgage"  branch and I will develop the mortgage feature of the calculator. 
I would be appreciated if you work on currency feature of the universal calculator and create the "feature_currency" branch. 
Please commit the changes and push the new branch to Git.
